From c493eef643ecb3a54c79083c51a8975f816ed9de Mon Sep 17 00:00:00 2001
From: Allan McRae <allan@archlinux.org>
Date: Tue, 28 Jun 2011 22:47:35 +1000
Subject: [PATCH] makepkg: fix removing symbolic link

The path was not being stripped from $file before prefixing with
$srcdir resulting in the attempted removal of a very weird
filename.

Signed-off-by: Allan McRae <allan@archlinux.org>
Signed-off-by: Dan McGee <dan@archlinux.org>
(cherry picked from commit e92905a2c8c14c7855e2841f44d3c139aa40844c)
---
 scripts/makepkg.sh.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/scripts/makepkg.sh.in b/scripts/makepkg.sh.in
index 3ace945..b0b0882 100644
--- a/scripts/makepkg.sh.in
+++ b/scripts/makepkg.sh.in
@@ -509,7 +509,7 @@ download_sources() {
 		local file=$(get_filepath "$netfile" || true)
 		if [[ -n "$file" ]]; then
 			msg2 "$(gettext "Found %s")" "${file##*/}"
-			rm -f "$srcdir/$file"
+			rm -f "$srcdir/${file##*/}"
 			ln -s "$file" "$srcdir/"
 			continue
 		fi
-- 
1.7.6

